boilerplates/fe-angular/README.md

```diff
@@ -0,0 +1,80 @@
+# fe-angular
+
+Angular quick starter project
+
+ ## Purpose of this quickstarter
+ 
+ This quick starter gives you a boilerplate code for new Angular project. This project contains basic setup for docker, jenkins, sonarqube and OpenShift. 
+
+ ## What files / architecture is generated?
+ ```
+.
+├── Jenkinsfile
+├── README.md
+├── angular.json
+├── docker
+│   ├── Dockerfile
+│   └── nginx.vh.default.conf.nginx
+├── e2e
+│   ├── protractor.conf.js
+│   ├── src
+│   │   ├── app.e2e-spec.ts
+│   │   └── app.po.ts
+│   └── tsconfig.e2e.json
+├── package.json
+├── sonar-project.properties
+├── src
+│   ├── app
+│   │   ├── app.component.html
+│   │   ├── app.component.scss
+│   │   ├── app.component.spec.ts
+│   │   ├── app.component.ts
+│   │   └── app.module.ts
+│   ├── assets
+│   ├── browserslist
+│   ├── environments
+│   │   ├── environment.prod.ts
+│   │   └── environment.ts
+│   ├── favicon.ico
+│   ├── index.html
+│   ├── karma.conf.js
+│   ├── main.ts
+│   ├── polyfills.ts
+│   ├── styles.scss
+│   ├── test.ts
+│   ├── tsconfig.app.json
+│   ├── tsconfig.spec.json
+│   └── tslint.json
+├── tsconfig.json
+└── tslint.json
+
+7 directories, 31 files
+```
+
+ ## Frameworks used
+
+This project is generated by Angular CLI. (https://cli.angular.io/)
+ * Angular CLI version `6.0.7`
+ * Angular version `^6.0.3`
+ * Typescript version - `2.8.1`
+
+ ## Usage - how do you start after you provisioned this quickstarter
+
+* Run command `npm install` in project directory to install npm packages.
+* Run `ng serve --open` command for dev server.
+
+ ## How this quickstarter is built through jenkins
+
+The Jenkinsfile is provisioned with this quick starter to ease CI/CD process.
+In Jenkinsfile, there are various stages
+  * stageBuild - Builds the application by running `yarn install`, `yarn build` command and copies output folder `dist` into `docker/dist` folder.
+
+  * stageUnitTest - Runs unit test cases by executing command `yarn test`.
+  * stageLint - Runs `ng lint` profiler by running command `npm run lint`.
+  * stageScanForSonarqube - Runs Sonarqube on current project
+  * stageStartOpenshiftBuild - Start building application on OpenShift
+  * stageDeployToOpenshift - Deploy output to OpenShift
```

Comment:
I would only comment on `stageBuild`, `stageUnitTest` and `stageLint` - all other stages are standard shared lib